<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>390</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Parametros:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>430</y>
      <width>531</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>223</x>
      <y>39</y>
      <width>541</width>
      <height>281</height>
     </rect>
    </property>
    <property name="plainText">
     <string>CARGUE UNA TRAYECTORIA EN EL SIGUIENTE FORMATO
NOMBRE
RECORRIDO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>520</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Ayuda</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Conexion">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Servidor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="auto_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>330</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Ejecutar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Modo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Cambiar Modo</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>60</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="IP_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>91</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="Puerto_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Puerto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>163</width>
      <height>348</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="Robot_2">
       <property name="text">
        <string>Robot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Motor_2">
       <property name="text">
        <string>Motor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Efector_2">
       <property name="text">
        <string>Efector</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Lineal_2">
       <property name="text">
        <string>Mov. Lineal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Circular_2">
       <property name="text">
        <string>Mov. Circular</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Actual">
       <property name="text">
        <string>Posicion Actual</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Origen">
       <property name="text">
        <string>Posicion de Descanso</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Reporte_2">
       <property name="text">
        <string>Reporte</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Trayectoria">
       <property name="text">
        <string>Cargar Taryectoria</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Eliminar">
       <property name="text">
        <string>Eliminar Trayectoria</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Enviar Taryectoria</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
